salt/config/__init__.py

```diff
@@ -1685,6 +1684,10 @@ def _read_conf_file(path):
     '''
     Read in a config file from a given path and process it into a dictionary
     '''
+    if (not salt.utils.validate.path.exists(path, quiet=os.path.basename(path) == '.saltrc')
```

Comment:
@vutny it is an _optional_ config file that can sit in home of user that executes e.g. minion and if something `echo "master: google.com"` to it, you will have rock-n-roll understanding why the heck your minion connects to google. :laughing: